### ISSUE TITLE
On inner pages, document has multiple elements referenced with ARIA with the same id attribute: modal-1-description. Need to ensure that every id attribute value used in ARIA and in labels is unique

### ISSUE THREAD

**Issue by shivaalipour** [GSA/marketplace-fedramp-gov#60]
Row 62

**Comment by shivaalipour** [comment]
@shivaalipour get more information about this

**Comment by echovanderhyden** [comment]
i think i see this one.  on line 84 of product-info-component there's a aria-describedby using "modal-1-description" when it should be modal-2-description.

**Comment by shivaalipour** [comment]
issue resolved
